src/Model/RedirectedURL.php

```diff
@@ -356,10 +306,56 @@ public function Link()
             return $link;
         }
 
-        // If we were unable to determine a Link, then we'll fallback to our $To value, in case there is one there
+        // If we were unable to determine a Link, then we'll fall back to our $To value, in case there is one there
         return $this->To;
     }
 
+    private function getRedirectCodeDefault(): int
+    {
+        $redirectCodeValue = 301;
+
+        $defaultRedirectCode = intval(Config::inst()->get(RedirectedURL::class, 'default_redirect_code'));
+
+        if ($defaultRedirectCode > 0) {
+            $redirectCodeValue = $defaultRedirectCode;
+        }
+
+        return $redirectCodeValue;
+    }
+
+    private function ensureBaseFromValidity(?string $fromBase): void
```

Comment:
```suggestion
    private function ensureFromBaseValidity(?string $fromBase): void
```
as we have `ensureFromQuerystringValidity` as well. Or update both to not contain the word From and just have `ensureBaseValidity` and `ensureQueryStringValidity`.

Reply:
Good spotting. Thanks.

I probably lean towards keeping the full field name as part of the method name. There are also some other "from" related methods, and I don't want to confuse those.